codegen/debug.go

```diff
@@ -598,6 +599,11 @@ func ExecWithFS(ctx context.Context, cln *client.Client, fs Filesystem, r io.Rea
 			securityMode = o.SecurityMode
 		case llbutil.NetworkOption:
 			netMode = o.NetMode
+		case llbutil.HostOption:
```

Comment:
Let's also add the equivalent to `ExecWithSolveErr`. It looks like `ExtraHosts` is exposed under `Meta` in `ExecOp`.

Reply:
nice catch,  forgot about that one. Fixed.